### ISSUE TITLE
A11y - Jupyter logo does not have enough contrast in dark mode

### ISSUE THREAD

**Issue by trallard** [jupyter/jupyter#620]
It seems that the introduction of a dark mode for the documentation at https://docs.jupyter.org/en/latest/ has introduced an undesired effect regarding the contrast between the logo in the navbar and the navbar colour (see image)

![Jupyter_Project_Documentation_—_Jupyter_Documentation_4_1_1_alpha_documentation](https://user-images.githubusercontent.com/23552331/181487516-8d1c14fe-e100-40dd-a85f-92e6e3e299aa.png)

I am unsure if the theme allows to specify a different logo for dark and light mode but this would be the best approach to resolve this issue
